packages/edit-site/src/components/app/index.js

```diff
@@ -40,6 +40,13 @@ export default function EditSiteApp( { reboot } ) {
 				{ ( { params } ) => {
 					const isListPage = getIsListPage( params );
 
+					// The existence of `'front-page` supersedes very other settings.
+					const homeTemplateType =
+						params.postId?.includes( 'front-page' ) ||
+						params.postId === homeTemplate?.postId
+							? 'site-editor'
+							: undefined;
```

Comment:
@kevin940726, do you think it's worth extracting this into a utility function like `getIsListPage`?

Reply:
Oops, only just saw this now. I don't have a preference, but usually I'd only do that if it's repeated multiple times.

Reply:
I do the same. I will extract it into a function when there's a need.